packages/migrate/src/MigrateEngine.ts

```diff
@@ -351,9 +351,10 @@ export class MigrateEngine {
             this.rejectAll(err)
             reject(err)
           }
+
           const engineMessage = this.lastError?.message || this.messages.join('\n')
           const handlePanic = () => {
-            const stackTrace = this.messages.join('\n')
+            const stackTrace = this.lastError?.backtrace || this.messages.join('\n')
```

Comment:
This may have the side effect of supporting proper stacktraces in `migrate` panics